packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -1457,10 +1465,10 @@ describe('TokenListController', () => {
           data: sampleSepoliaTokensChainCache,
         },
       });
+      controller.stopPollingByPollingToken(pollingToken);
```

Comment:
Nit: This test seems to do a lot. It's testing `startPollingByNetworkClientId`, but also `stopPollingByPollingToken`. I'm wondering if it's worth it to look at this test to see if it can be split up. I realize this is somewhat difficult, because in order to stop polling you have to start it somehow, so these methods seem linked. But I feel like this test would be hard to read for future maintainers. Nothing to do about this now, but something to think about as we increase how many controllers use this pattern.